<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LCFile</class>
 <widget class="QDialog" name="LCFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LCFile</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>561</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Input</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>54</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Input Text File with All the &quot;NLCD Classfication Types&quot; Present in the Domain</string>
    </property>
    <property name="text">
     <string>NLCD File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNLCD">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Input Text File with All the &quot;NLCD Classfication Types&quot; Present in the Domain</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNLCD">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>46</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Input Text File with All the &quot;NLCD Classfication Types&quot; Present in the Domain</string>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>103</y>
     <width>561</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>52</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Output will be saved to selected the .LC File</string>
    </property>
    <property name="text">
     <string>.LC File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLC">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>48</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Output will be saved to selected the .LC File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLC">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>45</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Output will be saved to selected the .LC File</string>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
  </widget>
  <widget class="Q3TextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>264</y>
     <width>561</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRun">
   <property name="geometry">
    <rect>
     <x>471</x>
     <y>220</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>RUN Generate LC Module</string>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Close!</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonHelp">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>220</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Help!</string>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
